web/src/pages/Profile/Votes/index.tsx

```
@@ -0,0 +1,8 @@
+import React from "react";
+
+interface IVotes {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)